LICENSE.md

```diff
@@ -1,13 +1,17 @@
 Copyright (C) 2018 - 2024. Licensed under the Creative Commons Attribution-NonCommercial 4.0 International Public License
 (https://creativecommons.org/licenses/by-nc/4.0/legalcode)
 
-Sharing is permitted — users may copy and redistribute the material in any medium or format. 
-Adaptation is permitted — users may remix, transform, and build upon the material. 
+Sharing is permitted — users may copy and redistribute the material in any medium or format.
+Adaptation is permitted — users may remix, transform, and build upon the material.
 Attribution — You must give appropriate credit, provide a link to the license, and indicate if changes were made.
 You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use.
-NonCommercial — You may not use the material, or derivitives thereof, for commercial purposes such as productization; resale; 
-incorporation into a product or service for resale; or other activities involving renumerartion.
-This shall not be construed to prohibit use by commericial entities provided the material, or its derivitives, 
-are not incorporated into a product or service that is sold or resold for monetary gain or renumeration.
-No additional restrictions — You may not apply legal terms or technological measures that legally restrict others
-from doing anything the license permits.
+NonCommercial — You may not use the material, or derivatives thereof, for commercial purposes such as productization; resale;
+incorporation into, or integration with, a product or service for resale; or other activities involving renumeration or trade.
+
+Usage by the following classes of entities is prohibited: security software product companies including but not limited EDR, NDR, XDR or SIEM product companies; search, database, data aggregation and / or data lake product companies; public cloud providers, SaaS and PaaS product, service or platform companies not otherwise specified.
```

Comment:
```suggestion
Usage by the following classes of entities is prohibited: security software product companies including but not limited to EDR, NDR, XDR or SIEM product companies; search, database, data aggregation and / or data lake product companies; public cloud providers, SaaS and PaaS product, service or platform companies not otherwise specified.
```